outbound.go

```diff
@@ -332,13 +307,22 @@ func cloneTags(tags map[string]string) map[string]string {
 // doneReading shuts down the message exchange for this call.
 // For outgoing calls, the last message is reading the call response.
 func (response *OutboundCallResponse) doneReading(unexpected error) {
-	now := response.GetTime()
-	response.AddAnnotationAt(AnnotationKeyClientReceive, now)
-	response.Report()
+	now := response.timeNow()
 
 	isSuccess := unexpected == nil && !response.ApplicationError()
 	lastAttempt := isSuccess || !response.requestState.HasRetries(unexpected)
 
+	// TODO how should this work with retries?
```

Comment:
This is a good question, I think previously we would report multiple separate spans. We should make sure it still makes sense -- I think separate spans for each attempt would be the best approach?